src/cookbooks/retention.md

```diff
@@ -1,225 +1,91 @@
 
-Note that we filter to profiles that were created in the first half of April so that we have sufficient time to observe 6 weeks of behavior. Now we can calculate retention!
+### Querying Smoot Usage Tables
 
-```python
-os_counts = (
-    ms
-    .groupby("os")
-    .agg(F.countDistinct("client_id").alias("total_clients"))
-)
-
-weekly_counts = (
-    ms
-    .groupby("period", "os")
-    .agg(F.countDistinct("client_id").alias("n_week_clients"))
-)
+For programatic access, the tables underlying GUD can be queried directly.  For example:
 
-retention_by_os = (
-    weekly_counts
-    .join(os_counts, on='os')
-    .withColumn("retention", F.col("n_week_clients") / F.col("total_clients"))
-    # Add a 95% confidence interval based on the normal approximation for a binomial distribution,
-    # p ± z * sqrt(p*(1-p)/n).
-    # The 95% CI spans the range `retention ± ci_95_semi_interval`.
-    .withColumn(
-      "ci_95_semi_interval",
-      F.lit(1.96) * F.sqrt(F.col("retention") * (F.lit(1) - F.col("retention")) / F.col("total_clients"))
-    )
-)
+```sql
+SELECT
+  `date`,
+  SAFE_DIVIDE(SUM(new_profile_active_in_week_1), SUM(new_profiles)) AS one_week_new_profile_retention,
+  SAFE_DIVIDE(SUM(active_in_weeks_0_and_1), SUM(active_in_week_0)) AS one_week_retention
+FROM `moz-fx-data-shared-prod.telemetry.smoot_usage_day_13`
+WHERE
+  usage = 'Any Firefox Desktop Activity'
+  AND country IN ('US', 'GB', 'CA', 'FR', 'DE')
+  AND `date` BETWEEN "2019-11-01" AND "2019-11-07"
+GROUP BY `date` ORDER BY `date`
 ```
 
-Peeking at 6-Week Retention
-
-```python
-retention_by_os.filter("period = 6").show()
-```
-
-```
-+----------+------+--------------+-------------+-------------------+--------------------+
-|        os|period|n_week_clients|total_clients| retention         | ci_95_semi_interval|
-+----------+------+--------------+-------------+-------------------+--------------------+
-|     Linux|     6|          1495|        22422|0.06667558647756668|0.003265266498407...|
-|    Darwin|     6|          1288|         4734|0.27207435572454586|0.012677372722376635|
-|Windows_NT|     6|         29024|       124872|0.23243000832852842|0.002342764476746...|
-+----------+------+--------------+-------------+-------------------+--------------------+
-
+### Querying Clients Daily Tables
+
+For more custom access, you use the clients_last_seen tables.  You can restrict to an arbitrary population of users by joining the `base` table below against a table containing the `client_id`s of interest.
+
+```sql
+CREATE TEMP FUNCTION
+  udf_active_n_weeks_ago(x INT64, n INT64)
+  RETURNS BOOLEAN
+  AS (
+    BIT_COUNT(x >> (7 * n) & (0x7F)) > 0
+  );
+CREATE TEMP FUNCTION
+  udf_bitpos( bits INT64 ) AS ( CAST(SAFE.LOG(bits & -bits, 2) AS INT64));
+
+WITH base AS (
+  SELECT
+    client_id,
+    submission_date,
```

Comment:
```suggestion
    DATE_SUB(submission_date, INTERVAL 13 DAY) AS date,
```

We should probably include the 13 day shift as part of this example query, and modify the final SELECT to reference `date = "2019-12-01".